core/commands/list_cloud_instances_command.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative 'base_command'
+require_relative '../services/aws_service'
+require_relative '../services/gcp_service'
+
+# Command shows list all active instances on Cloud Providers
+class ListCloudInstancesCommand < BaseCommand
+  HIDDEN_GCP_INSTANCES = %w[
```

Comment:
The black list should be configurable. Put it into the MDBCI configuration file.